zebra-chain/src/transaction/arbitrary.rs

```diff
@@ -236,6 +256,8 @@ impl Arbitrary for Transaction {
             NetworkUpgrade::Blossom | NetworkUpgrade::Heartwood | NetworkUpgrade::Canopy => {
                 Self::v4_strategy(ledger_state)
             }
+            // Blocked by #1823
+            //NetworkUpgrade::NU5 => Self::v5_strategy(ledger_state),
```

Comment:
NU5 supports v4 and v5 transactions:
> [Orchard proposal] The transaction version number MUST be 4 or 5.

Page 106 in https://raw.githubusercontent.com/daira/zips/orchard-circuit/protocol/orchard.pdf

```suggestion
            // Blocked by #1823 and #1824, see #1826
            //NetworkUpgrade::NU5 => prop_oneof!(Self::v4_strategy(ledger_state), Self::v5_strategy(ledger_state)),
```

Reply:
This code is currently unreachable, because the activation heights for `NU5` haven't been chosen yet.

Let's open a separate ticket to work out how to deal with this design issue. We might need to add a `network_upgrade_override` field, or something similar.

(Since the code is unreachable anyway, we don't need to wait for #1823 for this PR to merge.)

This follow-up is now #1826.